### PR TITLE
allow passing arbitary cl2 args

### PR DESCRIPTION
**Pull request by upodroid**
Required for https://github.com/kubernetes/kops/pull/16181

### CODE REVIEW

**Review by upodroid — COMMENTED** [review]
/retest 
/cc @dims @jbpratt

File: pkg/testers/clusterloader2/cl2.go [hunk 1]
```diff
@@ -44,6 +45,7 @@ type Tester struct {
 	Nodes                     int    `desc:"Number of nodes in the cluster. 0 will auto-detect schedulable nodes."`
 	EnablePrometheusServer    bool   `desc:"Whether to set-up the prometheus server in the cluster."`
 	PrometheusPvcStorageClass string `desc:"Storage class used with prometheus persistent volume claim."`
+	ExtraCL2Args              string `flag:"~extra-cl2-args" desc:"Additional arguments supported by clusterloader2 (https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/cmd/clusterloader.go)."`
```
Comment (dims) [inline pkg/testers/clusterloader2/cl2.go:48]: can we leave the flag as just `extra-args`?

Comment (upodroid) [reply pkg/testers/clusterloader2/cl2.go]: done

**Comment by upodroid** [comment]
@jbpratt ^

**Pull request merged**